src/index.js

```diff
@@ -3,7 +3,8 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const codecs = require('multicodec/src/base-table.json')
+const { baseTable } = require('multicodec/src/base-table.js')
```

Comment:
```suggestion
const { baseTable: codecs } = require('multicodec/src/base-table.js')
```

Reply:
Oh sweet, I was hoping there would be syntax for that. Nice!